examples/inverse/evoked_ers_source_power.py

```diff
@@ -90,23 +106,17 @@
 # See :ref:`ex-inverse-source-power` for more information about DICS.
 
 
-def _gen_dics(active_win, baseline_win, epochs):
-    freqs = np.logspace(np.log10(12), np.log10(30), 9)
-    csd = csd_morlet(epochs, freqs, tmin=-1, tmax=1.5, decim=20)
-    csd_baseline = csd_morlet(epochs, freqs, tmin=baseline_win[0],
-                              tmax=baseline_win[1], decim=20)
-    csd_ers = csd_morlet(epochs, freqs, tmin=active_win[0], tmax=active_win[1],
-                         decim=20)
+def _gen_dics(csd, ers_csd, baseline_csd, fwd):
     filters = make_dics(epochs.info, fwd, csd.mean(), pick_ori='max-power',
                         reduce_rank=True, real_filter=True, rank=rank)
-    stc_base, freqs = apply_dics_csd(csd_baseline.mean(), filters)
-    stc_act, freqs = apply_dics_csd(csd_ers.mean(), filters)
+    stc_base, freqs = apply_dics_csd(baseline_csd.mean(), filters)
+    stc_act, freqs = apply_dics_csd(ers_csd.mean(), filters)
```

Comment:
one reason I would rather use `csd_ers` rather than `ers_csd` is that when I type I prefer to autocomplete based on the type of object. Basically all variables that are of type csd start with csd_ so csd_<TAB> gives me what I want.